src/provider/utils/async.ts

```diff
@@ -1,10 +1,19 @@
 import {Signal} from 'openfin-service-signal';
 
 import {DeferredPromise} from '../common/DeferredPromise';
+import {Timeouts} from '../constants';
+
+export enum CollateApiCallResultsResult {
+    AnySuccess,
+    AllFailure,
+    Timeout,
+    NoCalls
```

Comment:
Can we use `CONST_CASE` for these? All enums outside of error codes (which we should look at as part of the review story) use const case.

Reply:
✅